src/CMakeLists.txt

```diff
@@ -35,9 +35,10 @@ add_library(${LIB_NAME} STATIC
 # cryptopp_pem comes from local source tree
 # linking PUBLIC as we want headers from all those libs be accessible to all clients
 target_link_libraries(${LIB_NAME}
-    PUBLIC
+    PRIVATE
     OpenSSL::SSL
+    spdlog::spdlog
     ${CRYPTOPP_LIBRARIES}
+    PUBLIC
```

Comment:
cryptopp_pem needs to be public ?

Reply:
Yes, the PEM component is also used somewhere in librpc client code. Since I extracted cryptopp_pem locally I do not want to copy sources to librpc.
This should be removed once @TomaszAugustyn is done with migration of PEM handling to OpenSSL.

Reply:
Hm, it shouldn't be used by librpc at all it should be used only be sdk, if that would be the case I guess we should link it directly in librpc, but I am 99.9% we don't use it. Actually we decided to leave cryptopp_pem now, maybe if it will be a need we remove it in the future :P